docs/v3.x/concepts/models.md

```diff
@@ -796,7 +796,7 @@ xhr.send(
 
 ## Dynamic Zone
 
-Dynamic Zone field let your create flexible space to content based on a component list.
+Dynamic Zone fields let you create flexible space, to compose content based on a component list, inside.
```

Comment:
The structure of this sentence doesn't entirely make sense, the commas create breaks that disrupt the structure.

I'd say something like:

>Dynamic Zone fields let you create a flexible space to compose content based on a component list 

That being said I can't honestly name any good examples of a useful purpose for DZs 🤔

Reply:
The purpose of a Dynamic Zone is to have mixed content types in an array, that upon display, can be iterated over. We’re using it to do page composition of, say, a title, a paragraph, a FAQ, another paragraph, etc.